platforms/ucp/templates/handlers/bfaa-transition-video-handler.ts

```diff
@@ -0,0 +1,49 @@
+import {
+	AdSlot,
+	DomManipulator,
+	Porvata4Player,
+	TEMPLATE,
+	TemplateStateHandler,
+	TemplateTransition,
+	UapParams,
+} from '@wikia/ad-engine';
+import { Inject, Injectable } from '@wikia/dependency-injection';
+import { from, Observable, Subject } from 'rxjs';
+import { take, takeUntil, tap } from 'rxjs/operators';
+import { BfaaVideoHelper } from '../helpers/bfaa-video-helper';
+import { BfaaContext } from './bfaa-context';
+
+@Injectable()
+export class BfaaTransitionVideoHandler implements TemplateStateHandler {
+	private unsubscribe$ = new Subject<void>();
+	private manipulator = new DomManipulator();
+	private helper: BfaaVideoHelper;
+
+	constructor(
+		@Inject(TEMPLATE.PARAMS) private params: UapParams,
+		@Inject(TEMPLATE.SLOT) private adSlot: AdSlot,
+		@Inject(TEMPLATE.CONTEXT) private context: BfaaContext,
+	) {
+		this.helper = new BfaaVideoHelper(this.manipulator, this.params, this.adSlot);
+	}
+
+	async onEnter(transition: TemplateTransition<'transition'>): Promise<void> {
```

Comment:
```suggestion
	async onEnter(): Promise<void> {
```

we do not transition anywhere in this handler